src/webob/acceptparse.py

```diff
@@ -1014,619 +822,97 @@ def acceptable_offers(self, offers):
 
     def best_match(self, offers, default_match=None):
         """
-        Return the best match from the sequence of media type `offers`.
+        Return the best match from the sequence of media type ``offers``.
 
-        .. warning::
+        This is a thin wrapper around :meth:`.acceptable_offers` that makes
+        usage more convenient for typical use-cases where you just want
+        to know the client's most preferred match.
 
-           This is currently maintained for backward compatibility, and will be
-           deprecated in the future.
+        :param offers:
+            (iterable)
 
-           :meth:`AcceptValidHeader.best_match` uses its own algorithm (one not
-           specified in :rfc:`RFC 7231 <7231>`) to determine what is a best
-           match. The algorithm has many issues, and does not conform to
-           :rfc:`RFC 7231 <7231>`.
+            | Each item in the iterable must be a ``str`` media type and
```

Comment:
"iterable" should be "sequence", as explained [here](https://github.com/Pylons/webob/pull/460/files#r1531444937).